datacite/version.py

```diff
@@ -18,4 +18,4 @@
 # Do not change the format of this next line. Doing so risks breaking
 # setup.py and docs/conf.py
 
-__version__ = "0.1.0.dev20140223"
+__version__ = "0.1.1.dev20150225"
```

Comment:
> 0.1.0

(Oh, I see this is because you added also post-release bump commit.  It is better to do it afterwards, not in the same PR, since it's easier to tag release and upload in that way.)